proxy/src/main/java/org/apache/rocketmq/proxy/ProxyStartup.java

```diff
@@ -173,7 +174,9 @@ protected static MessagingProcessor createMessagingProcessor() {
             PROXY_START_AND_SHUTDOWN.appendStartAndShutdown(proxyMetricsManager);
         } else if (ProxyMode.isLocalMode(proxyModeStr)) {
             BrokerController brokerController = createBrokerController();
-            ProxyMetricsManager.initLocalMode(brokerController.getBrokerMetricsManager(), ConfigurationManager.getProxyConfig());
+            if (brokerController.getBrokerConfig().getMetricsExporterType() != BrokerConfig.MetricsExporterType.DISABLE) {
```

Comment:
It seems to me that checking this config in ProxyMetricsManager#initLocalMode and ProxyMetricsManager#start should be better.